ontologies/CALM_single_MS.md

```diff
@@ -0,0 +1,24 @@
+
+# JSON examples: a free-standing manuscript described in CALM
+
+Comment, lines 8-13
+CALM contains both RefNo (to build the hierarchy) and AltRefNo (to show to the user).
+Both will presumably be needed.  At present each maps simply to identifier (=sourceIdentifier in the .ttl files).  
+How to distinguish?
+Also occurs with Item, below
+
+Comment, lines 47-52
+CALM contains both RefNo (to build the hierarchy) and AltRefNo (to show to the user).
+Both will presumably be needed.  At present each maps simply to identifier (=sourceIdentifier in the .ttl files).  
+How to distinguish?
```

Comment:
If I go to the library website right now, do I see both of them? How are they named there?